highfidelity/hifi
# CR for Job #20774 - Allowance for negative zone area space within the Domain-Server web portal.

Code Review for Job #20774 - Workitem available at https://www.worklist.net/20774

## Testnotes

- Launch the High Fidelity Server
- Go To the settings page
- Create an audio zone in the negative space of a domain   eg.    ( xyz -6000 to -3000 ) 
- Try connecting the zone with a positve space audio zone
- The settings should persist after restarting the server


#### Note: This PR will make the old notation of audio-zones invalid, please recreate them
